.github/workflows/dev-build.yml

```diff
@@ -126,6 +126,7 @@ jobs:
           # This removes the ability to sign in
           REACT_APP_DISABLE_AUTH: true
 
+        shell: bash
```

Comment:
isn't it already?

Reply:
Ah, after some more digging, I see that it uses the default shell of `ubuntu-latest`, which is `bash`. I'll remove these. Thanks @peterbe!